.eslintrc

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "@segment/eslint-config/browser/legacy"
```

Comment:
Do you think we still need to keep the old config around?

Reply:
I think we should, `browser/legacy` ensures we are only using ES3 compatible code which we can't really guarantee otherwise given we don't use any transpilers

Reply:
Gotcha, that makes sense - we definitely need that 🙏